overlord/snapshotstate/snapshotstate.go

```diff
@@ -69,6 +78,75 @@ func allActiveSnapNames(st *state.State) ([]string, error) {
 	return names, nil
 }
 
+func automaticSnapshotExpiration(st *state.State) time.Duration {
+	// TODO: get from config
+	return defaultAutomaticSnapshotExpiration
+}
+
+// saveExpiration saves expiration date of the given snapshot set in the state.
```

Comment:
Is it a snapshot that is set in the state or a snapshot set that is in the state? (where is the comma?)

```suggestion
// saveExpiration saves expiration date of the given snapshot set, in the state.
```